# Robot name goes here
mini_robot:
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller 1
  rear_drive_controller:
    type: velocity_controllers/JointVelocityController 
    joint: "rear_drive_joint"
    pid : {p: 100.0 , i: 0.0 , d: 0.0}

  # Controller 2
  left_front_steer_controller:
    type: position_controllers/JointPositionController 
    joint: "left_front_steer_joint"
    pid : {p: 10.0 , i: 0.0 , d: 0.0}

  # Controller 3
  right_front_steer_controller:
    type: position_controllers/JointPositionController 
    joint: "right_front_steer_joint"
    pid : {p: 10.0 , i: 0.0 , d: 0.0}



 
